session.go

```diff
@@ -172,6 +172,10 @@ func Sessioner(opts ...Options) flamego.Handler {
 		sid := opt.ReadIDFunc(c.Request().Request)
 		sess, created, err := mgr.load(c.Request().Request, sid, opt.IDLength)
 		if err != nil {
+			if errors.Cause(err) == context.Canceled {
+				c.ResponseWriter().WriteHeader(0)
```

Comment:
cc @unknwon

Reply:
If after the user closes the connection, the other middlewares should continue to execute, but because the ctx has been canceled when loading, the loaded session data is empty, in this case, the Store in line 190 should not be executed? 
In this case, how do we handle the flash message? Or can we use a new ctx for session loading?